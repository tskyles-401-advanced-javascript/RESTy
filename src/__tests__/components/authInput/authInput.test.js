import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import React from "react";
import renderer from "react-test-renderer";
import AuthInput from "../../../components/authInput/authInput";
Enzyme.configure({ adapter: new Adapter() });

describe("< />", () => {
  it("is alive on start", () => {
    let app = shallow(<AuthInput />);
    expect(app.find("div").exists()).toBeTruthy();
    expect(app.find("form").exists()).toBeTruthy();
    expect(app.find("#name").exists()).toBeTruthy();
    expect(app.find("#pass").exists()).toBeTruthy();

  });

  it("renders correctly", () => {
    const tree = renderer.create(<AuthInput />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});