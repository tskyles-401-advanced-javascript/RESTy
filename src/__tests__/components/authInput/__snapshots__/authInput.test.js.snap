// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`< /> renders correctly 1`] = `
<div>
  <h4>
    Basic Auth
  </h4>
  <form>
    <input
      id="name"
      placeholder="username"
    />
    <input
      id="pass"
      placeholder="password"
    />
    <h4>
      Bearer Token
    </h4>
    <input
      placeholder="token"
    />
  </form>
</div>
`;
